packages/augur-core/source/contracts/Augur.sol

```diff
@@ -85,7 +85,7 @@ contract Augur is IAugur {
     event DisputeWindowCreated(address indexed universe, address disputeWindow, uint256 startTime, uint256 endTime, uint256 id, bool initial);
     event InitialReporterTransferred(address indexed universe, address indexed market, address from, address to);
     event MarketTransferred(address indexed universe, address indexed market, address from, address to);
-    event MarketVolumeChanged(address indexed universe, address indexed market, uint256 volume, uint256[] outcomeVolumes);
```

Comment:
If it's necessary then I can sum the numbers in the contract. However can you describe how it significantly slows down syncing? It should be quite quick to sum up 8 numbers. Relative to the latency of a network transfer or disk write it should be trivial, so I have trouble understanding how it would add substantially to sync time.

Reply:
Its not the biggest impact, but we generally err on the side of reducing sync time over small gas costs. If during a sync there are 10000 active markets this means we need to do a looping sum for every market to derive each market's volume during syncing where every ms is additional time before the UI is ready.